scripts/install

```diff
@@ -20,166 +20,193 @@ ynh_abort_if_errors
-# SPECIFIC SETUP
 #=================================================
 # CONFIGURE
 #=================================================
-ynh_configure config.php "$final_path/application/config/config.php"
+ynh_script_progression --message="Configuring Limesurvey..." --weight=1
+
+ynh_add_config --template="../conf/config.php" --destination="$final_path/application/config/config.php"
 
 #=================================================
 # INSTALL
 #=================================================
+
 ls_cli=$final_path/application/commands/console.php
+
 fullname=$(ynh_user_get_info "$admin" "fullname")
 mail=$(ynh_user_get_info "$admin" "mail")
 
 # Permission should be correctly set before to do this
-ynh_exec_as "$app" php $ls_cli install "$admin" "$(ynh_string_random 24)" "$fullname" "$mail"
-
-#=================================================
-# STORE THE CHECKSUM OF THE CONFIG FILE
-#=================================================
+ynh_exec_as "$app" php $ls_cli install "$admin" "$password" "$fullname" "$mail"
 
-# Calculate and store the config file checksum into the app settings
-ynh_store_file_checksum "$final_path/application/config/config.php"
+# Permission should be correctly set before to do this
+#ynh_exec_as "$app" php $ls_cli install "$admin" "$(ynh_string_random 24)" "$fullname" "$mail"
 
 #=================================================
 # LOAD SQL SPECIFIC CONFIG
 #=================================================
-ynh_configure data.sql ./data.sql
-mysql -u $db_user -p$db_pwd $db_user < ./data.sql
+ynh_add_config --template="../conf/data.sql" --destination="./data.sql"
 
-#=================================================
-# Add nice themes
-#=================================================
-#ynh_setup_source "$final_path/upload/templates/libreform" libreform
-#ynh_setup_source "$final_path/upload/templates/librepoll" librepoll
+ynh_mysql_connect_as --user="$db_user" --password="$db_pwd" --database="$db_name" < ./data.sql
 
+ynh_secure_remove --file=./data.sql
 
 #=================================================
-# GENERIC FINALISATION
-#=================================================
-# SECURING FILES AND DIRECTORIES
+# Add nice themes
 #=================================================
-ynh_script_progression --message="Apply permissions..." --time --weight=1
-# Set permissions
-set_permissions
+
+#ynh_setup_source "$datadir/upload/templates/libreform" libreform
+#ynh_setup_source "$final_path/upload/templates/librepoll" librepoll
 
 #=================================================
 # SETUP FAIL2BAN
 #=================================================
-# ynh_script_progression --message="Configuring fail2ban..." --time --weight=1
-#ynh_add_fail2ban_config "/var/log/nginx/${domain}-error.log" "PHP message: Leed: wrong login for .* client: <HOST>" 5
+#ynh_script_progression --message="Configuring fail2ban..." --weight=1
+
+# No message in logs and the apps contains already a 10 minutes ban policy
```

Comment:
```suggestion
```

(No need to keep commented stuff)

Reply:
```suggestion
# No message in logs and the apps contains already a 10 minutes ban policy
```
I keep the comment to explain why it's not a problem to have no fail2ban conf. For once I take the time to write a comment :p